src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int COLOR_COUNT = Color.values().length;
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(COLOR_COUNT);
+        Color color = Color.values()[index];
+
+        return color.toString().toLowerCase();
```

Comment:
Instead of using `toString().toLowerCase()`, you should use the `name()` method to get the string representation of the enum constants as per the checklist. The `name()` method will return the enum constant in exactly the way it's declared, thus preserving the naming convention. Consider returning `color.name()` instead.